ansible-ipi-install/roles/set-deployment-facts/tasks/main.yml

```diff
@@ -17,3 +19,4 @@
     - lab_ipmi_user
     - lab_ipmi_password
     - ocp_deploying_node_content
+    - provisioner_type
```

Comment:
Don't you also need masters_prov_nic here?

Reply:
@smalleni No, if using 1029U model as masters, they have to use same modes as prov OR set this var in host file. https://github.com/redhat-performance/JetSki/blob/master/ansible-ipi-install/inventory/jetski/hosts#L125